RMStore.podspec

```diff
@@ -2,7 +2,7 @@ Pod::Spec.new do |s|
   s.name = 'RMStore'
   s.version = '0.7.1'
   s.license = 'Apache 2.0'
-  s.summary = 'A lightweight iOS library for In-App Purchases that adds blocks and notifications to StoreKit, plus verification, persistence and downloads.'
```

Comment:
I remember having to shorten the description for some reason.

In any case, would you mind taking this change out of the PR?